.github/workflows/licence-checker.yml

```diff
@@ -27,6 +29,12 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
 
+      - name: use staged maven repo settings
+        if: ${{ env.USE_STAGE == 'true' }}
+        run: |
+          cp $HOME/.m2/settings.xml /tmp/settings.xml
+          mv -vf .github/configs/settings.xml $HOME/.m2/settings.xml
+
```

Comment:
seems no need in license-check?

Reply:
I also feel that it is not necessary, but it's in this [PR](https://github.com/apache/incubator-hugegraph-toolchain/pull/536/files#diff-f5ad8c4519812f1e04dd23ed7c7421ab44abb64df10ba188732b21e7d52a23af). I will remove it, think you!

Reply:
> PR

Get the problem, and hubble ci do need it (now put it in a wrong place)
<img width="1281" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/f27cf83a-ffea-497a-92dd-78c5917f74c1">

Reply:
Seems changed wrong place?
<img width="1121" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/67c6670f-c777-4f93-b9f7-90fa0383977f">

Reply:
I will fix it ，think you！